spec/fixtures/files/test.tsv

```diff
@@ -0,0 +1,7 @@
+UID/tName/tStart date/tEnd date/tCity/tCountry/tRegion/tWebsite/tNotes
```

Comment:
Is this what an actual Google export looks like? The  `/t` isn't a tab escape. `\t` would be. That's why I am curious.